### DISCUSSION TITLE
addEventListener event.pageX pageY on Nuxt3

### DISCUSSION THREAD

**Discussion by zupcode** [nuxt/nuxt#19139] [Questions]
Hello everybody,

I've made a project in Nuxt2 with a page that uses JS `event.pageX` and `event.pageY` to catch the mouse position, and it works fine.
Something like that :

```
<template>
  <div>
    <div @mousemove="move()"></div>
    <div id="result"></div>
  </div>
</template>

<script>
export default {
  methods: {
    move(event) {
     document.getElementById('result').innerHTML = event.pageX + '/' + event.pageY
    }
  }
}
</script>
```

But this code doesn't work in Nuxt3...
I've tried many ways but I can't figure out how it could work or something similar.

Does someone have any idea or a suggestion ?
Thanks !

**Comment by abdul-alhasany** [comment]
Do you see an errors in console?

I think it is not working because you are calling the move function. Try changing your code to this:
```vue
<template>
  <div>
    <div @mousemove="move"></div> <!-- remove () -->
    <div id="result"></div>
  </div>
</template>

<script>
export default {
  methods: {
    move(event) {
     document.getElementById('result').innerHTML = event.pageX + '/' + event.pageY
    }
  }
}
</script>
```

**Comment by zupcode** [comment]
OK... Thanks a lot !
Just this little thing... shame on me...:)

But why it works with `()` in Nuxt2 and not in Nuxt3...??? Mystery
Thank you !

**Reply by abdul-alhasany** [reply]
Don't worry, it happens to all of us.

I am not sure how it was working in Nuxt2. I was actually surprised when you said it did. () calls the function, and essentially you are calling it without passing `$event` paramter, then you are using event in function definition to get x and y.  That can not work as nothing is being passed.

**Comment by zupcode** [comment]
Of course, it's logical, I should have thought about it...
